iroh-resolver/src/unixfs.rs

```diff
@@ -279,6 +279,19 @@ impl UnixfsNode {
         }
     }
 
+    /// Returns the blocksizes of the links
+    /// Should only be set for File
+    pub fn blocksizes(&self) -> Option<&[u64]> {
```

Comment:
I woulld suggest returning this always, as we might be reading data not produced by us.